sample_configs/README.md

```diff
@@ -0,0 +1,14 @@
+## Sample Configurations
+
+### Notes 
+
+This folder contains sample configurations for basic data pipelines using Bender. It is assumed you have a basic understanding of Lambda and the IAM permissions model. A good starting guide is [Are You a First-time User of AWS Lambda?](http://docs.aws.amazon.com/lambda/latest/dg/welcome.html)
+
+It is important to note that the pipelines outlined are functional but rudimentary and not intended for for production use. Consult the following resources for building pipelines using Lambda.
```

Comment:
`Consult the following resources for building [production] pipelines using Lambda.`